modules/editorial-metadata/editorial-metadata.php

```diff
@@ -327,1114 +328,1203 @@ function register_taxonomy() {
-		
-		$terms = $this->get_editorial_metadata_terms();
-		// We're looking for the proper term to display its saved value
-		foreach( $terms as $term ) {
-			$key = $this->module->slug . '-' . $term->slug;
-			if ( $column_name != $key )
-				continue;
-
-			$current_metadata = $this->get_postmeta_value( $term, $post_id );
-			echo $this->generate_editorial_metadata_term_output( $term, $current_metadata );
-		}
-		
-	}
-	
-	/**
-	 * If the Edit Flow Calendar is enabled, add viewable Editorial Metadata terms
-	 *
-	 * @since 0.7
-	 * @uses apply_filters( 'ef_calendar_item_information_fields' )
-	 *
-	 * @param array $calendar_fields Additional data fields to include on the calendar
-	 * @param int $post_id Unique ID for the post data we're building
-	 * @return array $calendar_fields Calendar fields with our viewable Editorial Metadata added
-	 */
-	function filter_calendar_item_fields( $calendar_fields, $post_id ) {
+			// or that the term name doesn't map to an existing term's slug
+			$search_term = $this->get_editorial_metadata_term_by( 'slug', sanitize_title( $metadata_name ) );
+			if ( is_object( $search_term ) && $search_term->term_id != $existing_term->term_id ) {
+				$change_error = new WP_Error( 'invalid', esc_html__( 'Name conflicts with slug for another term. Please choose again.', 'edit-flow' ) );
+				die( esc_html( $change_error->get_error_message() ) );
+			}
 
-		
-		// Make sure we respect which post type we're on
-		if ( !in_array( get_post_type( $post_id ), $this->get_post_types_for_module( $this->module ) ) )
-			return $calendar_fields;
-			
-		$terms = $this->get_editorial_metadata_terms( array( 'viewable' => true ) );
-		
-		foreach( $terms as $term ) {
-			$key = $this->module->slug . '-' . $term->slug;
-
-			// Default values
-			$current_metadata = $this->get_postmeta_value( $term, $post_id );
-			$term_data = array(
-				'label' => $term->name,
-				'value' => $this->generate_editorial_metadata_term_output( $term, $current_metadata ),
+			// Prepare the term name and description for saving
+			$args = array(
+				'name' => $metadata_name,
+				'description' => $metadata_description,
 			);
-			$term_data['editable'] = true;
-			$term_data['type'] = $term->type;
-			$calendar_fields[$key] = $term_data;
+			$return = $this->update_editorial_metadata_term( $existing_term->term_id, $args );
+			if ( ! is_wp_error( $return ) ) {
+				set_current_screen( 'edit-editorial-metadata' );
+				$wp_list_table = new EF_Editorial_Metadata_List_Table();
+				$wp_list_table->prepare_items();
+				echo wp_kses_post( $wp_list_table->single_row( $return ) );
+				die();
+			} else {
+				/* Translators: 1: the name of the term that could not be found */
+				$change_error = new WP_Error( 'invalid', wp_kses( sprintf( __( 'Could not update the term: <strong>%s</strong>', 'edit-flow' ), $metadata_name ), 'strong' ) );
+				die( esc_html( $change_error->get_error_message() ) );
```

Comment:
This'll output a literal `<strong>` insteag of the HTML tag due to double-escaping. Can move the `wp_kses()` call into `die()` like this other comment: https://github.com/Automattic/Edit-Flow/pull/734/files/0c6d169d72a8d91145d034d987371c97447b6869#r1655519700

Reply:
Done